Header header
uint32 custom_mode
uint16 type
uint16 autopilot
uint16 base_mode
uint16 system_status
uint16 mavlink_version
